ush/retrieve_data.py

```diff
@@ -984,7 +1054,7 @@ def parse_args(argv):
     )
     parser.add_argument(
         "--file_type",
-        choices=("grib2", "nemsio", "netcdf"),
+        choices=("grib2", "nemsio", "netcdf", "bufr", "prepbufr"),
```

Comment:
I'm not sure these got used anywhere.

Reply:
prepbufr and tcvitals added here to accomodate retrievals of the GFS prepbufr and tcvitals (called with the --file_type arg when invoking retrieve_data.py)